### PR TITLE
MAISTRA-1453 Collect jaeger and kiali resources in control plane name…

### PR DESCRIPTION
**Pull request by dgn**
…spaces

### CODE REVIEW

File: gather_istio [hunk 1]
```diff
@@ -17,6 +17,10 @@
 MUST_GATHER=${MUST_GATHER:-/usr/bin/openshift-must-gather}
 BASE_COLLECTION_PATH="/must-gather"
 
+# The following CRDs are not managed by the Maistra operator but it
+# will create instances of them.
+DEPENDENCY_CRDS="jaegers.jaegertracing.io kialis.kiali.io"
```
Comment (jwendell) [inline gather_istio]: I'd call this just `CR`, not `CRD`, to avoid confusion.

Comment (dgn) [reply gather_istio]: I agree, I changed it

File: gather_istio [hunk 2]
```diff
@@ -143,6 +147,9 @@ function main() {
     local members=$(getMembers ${cp})
       resources+="$(addResourcePrefix ns ${members})"
       getSynchronization ${cp}
+      for crd in ${DEPENDENCY_CRS}; do
```
Comment (jwendell) [inline gather_istio]: `cr` here too :)

**Review by brian-avery — APPROVED** [review]

**Review by jwendell — APPROVED** [review]
thanks

**Pull request merged**